### PR TITLE
Update echo messages in tar installer

### LINKED ISSUE #917: [Bug]: OpenSearch dist `opensearch-tar-install.sh` does not work
### Describe the bug

Not sure if it's just my local environment setup. Seems like I have to set an environment variable.

When trying to run `opensearch-tar-install.sh` I get the following message:

```
OpenSearch Security Demo Installer
 ** Warning: Do not use on production or public reachable systems **
Basedir: /home/ubuntu/releases/opensearch-1.2.0
OpenSearch install type: .tar.gz on DISTRIB_ID=Ubuntu
OpenSearch config dir: /home/ubuntu/releases/opensearch-1.2.0/config
OpenSearch config file: /home/ubuntu/releases/opensearch-1.2.0/config/opensearch.yml
OpenSearch bin dir: /home/ubuntu/releases/opensearch-1.2.0/bin
OpenSearch plugins dir: /home/ubuntu/releases/opensearch-1.2.0/plugins
OpenSearch lib dir: /home/ubuntu/releases/opensearch-1.2.0/lib
Detected OpenSearch Version: x-content-1.2.0
Detected OpenSearch Security Version: 1.2.0.0
/home/ubuntu/releases/opensearch-1.2.0/config/opensearch.yml seems to be already configured for Security. Quit.
done security
done plugins
Checking kNN library
TEST FAILED OR FILE NOT EXIST /home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libKNNIndexV2_0_11.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_common.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_faiss.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_nmslib.so
KNN lib path not found, set new path
:/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib
Exception in thread "main" java.lang.NoSuchFieldError: LUCENE_8_10_1
        at org.opensearch.Version.<clinit>(Version.java:77)
        at org.opensearch.bootstrap.Bootstrap.<init>(Bootstrap.java:98)
        at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:366)
        at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178)
        at org.opensearch.bootstrap.OpenSearch.execute(OpenSearch.java:169)
        at org.opensearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:100)
        at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138)
        at org.opensearch.cli.Command.main(Command.java:101)
        at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:135)
        at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:101)
```

Looks like file was renamed or removed.

### To reproduce

* `wget -c https://ci.opensearch.org/ci/dbc/bundle-build/1.2.0/941/linux/x64/dist/opensearch-1.2.0-linux-x64.tar.gz -O - | tar -xz`
* `./opensearch-1.2.0/opensearch-tar-install.sh`

### Expected behavior

`.opensearch-tar-install.sh` to work and setup my cluster.

### Screenshots

n/a

### Host / Environment

Ubuntu

### Additional context

The assemble workflow takes this exact script:
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/legacy/tar/linux/opensearch-tar-install.sh

and adds it to the `dist` folder. If there is updates required the linked script should be updated.

### Relevant log output

```shell
Checking kNN library
TEST FAILED OR FILE NOT EXIST /home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libKNNIndexV2_0_11.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_common.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_faiss.so
/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib/libopensearchknn_nmslib.so
KNN lib path not found, set new path
:/home/ubuntu/releases/opensearch-1.2.0/plugins/opensearch-knn/knnlib
Exception in thread "main" java.lang.NoSuchFieldError: LUCENE_8_10_1
        at org.opensearch.Version.<clinit>(Version.java:77)
        at org.opensearch.bootstrap.Bootstrap.<init>(Bootstrap.java:98)
        at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:366)
        at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178)
        at org.opensearch.bootstrap.OpenSearch.execute(OpenSearch.java:169)
        at org.opensearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:100)
        at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138)
        at org.opensearch.cli.Command.main(Command.java:101)
        at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:135)
        at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:101)
```

### PR DESCRIPTION
**Pull request by jmazanec15**
Signed-off-by: John Mazanec <jmazane@amazon.com>

### Description
This PR cleans up some of the echo messages around the k-NN library to give more accurate information on tar install. I removed the library check because that seems unnecessary. Additionally, I added a message to indicate to the user that OpenSearch is starting.
 
### Issues Resolved
#917 
 
### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: scripts/legacy/tar/linux/opensearch-tar-install.sh [hunk 1]
```diff
@@ -40,33 +40,25 @@ if ! grep -q '## OpenSearch Performance Analyzer' $OPENSEARCH_HOME/config/jvm.op
...
-    echo "FILE EXISTS $FILE"
-else
-    echo "TEST FAILED OR FILE NOT EXIST $FILE"
-fi
-
 ##Set KNN Dylib Path for macOS and *nix systems
 if echo "$OSTYPE" | grep -qi "darwin"; then
     if echo "$JAVA_LIBRARY_PATH" | grep -q "$KNN_LIB_DIR"; then
-        echo "KNN lib path has been set"
+        echo "k-NN libraries found in JAVA_LIBRARY_PATH"
```
Comment (peternied) [inline scripts/legacy/tar/linux/opensearch-tar-install.sh:46]: Could you update the messages to include the paths that were used after the variable name?  Would that help with debugging?

**Review by gaiksaya — APPROVED** [review]
Thank you for this. It was long due :)

**Review by peternied — APPROVED** [review]

File: scripts/legacy/tar/linux/opensearch-tar-install.sh (hunk 1 above)
Comment (jmazanec15) [reply scripts/legacy/tar/linux/opensearch-tar-install.sh:46]: In my opinion, I think if it is found, just saying so is okay.

If it is not found, echo'ing the updated path could be useful. Currently, we do echo the path. But, do you mean update it to something like this? 
```
        echo "k-NN libraries not found in JAVA_LIBRARY_PATH. Updating path to: $JAVA_LIBRARY_PATH."
```

**Comment by gaiksaya** [comment]
Why is the merging blocked on this one? 🤔

**Comment by peterzhuamazon** [comment]
@jmazanec15 you need to setup signature with GitHub, someone has turned on the verification of signature in this repo.
https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key

**Pull request merged**